lxd/instance/drivers/driver_qemu.go

```diff
@@ -1809,6 +1872,19 @@ func (vm *qemu) generateQemuConfigFile(busName string, devConfs []*deviceConfig.
 		return "", err
 	}
 
+	devBus, devAddr, multi = bus.allocate(busFunctionGroup9p)
+	err = qemuVirtioFSDriveConfig.Execute(sb, map[string]interface{}{
+		"bus":           bus.name,
+		"devBus":        devBus,
+		"devAddr":       devAddr,
+		"multifunction": multi,
+
+		"path": filepath.Join(vm.Path(), "virtiofs.sock"),
```

Comment:
Since it's not a device and doesn't need to be persistent, probably use `filepath.Join(vm.LogPath(), "virtiofs.config.sock")` instead?